tests/plugins/UM3NetworkPrinting/TestSendMaterialJob.py

```diff
@@ -0,0 +1,314 @@
+# Copyright (c) 2018 Ultimaker B.V.
+# Cura is released under the terms of the LGPLv3 or higher.
+
```

Comment:
Plugins need to contain their own tests (eg; see CuraProfileReader or OBJReader for instance)

Reply:
I think this was already moved but not pushed yet :(

Also, the tests in plugins are not ran by the CI under Linux (as they are not covered in pytest.ini), that was fixed on Marijns machine as well, including some things that fell over because of it (incorrect import paths in the version upgrader tests).

Reply:
Should the code change for that? I see that the other test use a different root for imports.

Reply:
Nope, sort of resolved now by having a custom pycharm setup for the tests.